src/applications/gi/tests/e2e/vet-tec-helpers.js

```diff
@@ -36,7 +35,7 @@ const verifySearchResults = (client, results = vetTecSearchResults) => {
     const id = `#search-result-${result.attributes.facility_code}-${
       result.attributes.description
     }`;
-    client.waitForElementVisible(UtilHelpers.createId(id), Timeouts.normal);
+    client.waitForElementVisible(GiHelpers.createId(id), Timeouts.normal);
```

Comment:
Once `import environment from 'platform/utilities/environment';` is removed from `src/applications/gi/utils/helpers.js` changes can be reverted